main_test.go

```diff
@@ -190,10 +196,10 @@ func TestInitOrReloadObsctlConfig(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			dir := t.TempDir()
 			testutil.Ok(t, os.Setenv("OBSCTL_CONFIG_PATH", path.Join(dir, "obsctl", "config.json")))
-			testutil.Ok(t, os.Setenv("MANAGED_TENANTS", tc.tenants))
 
 			// Handle test cases with pre-exisiting config.
 			if tc.prexistingConfig != nil {
+				o.managedTenants = tc.tenants
```

Comment:
maybe we could add a test case with where `tc.tenants` has a non-expected format (e.g. not comma separated')?

Reply:
So if it's not comma-separated, we consider it as the name of a single tenant. There isn't really an error case here to test 🙂

Reply:
I see - in case is then without a comma, is the input validated in any way?

Reply:
sounds good then!